### PR TITLE
com.coeck_studios.Dinonuggys-Journey: init at 0.59.2

### PR DESCRIPTION
**Pull request by papojari**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by papojari** [comment]
Note that the actual domain is `coeck-studios.com` not `coeck_studios.com`.

**Comment by hfiguiere** [comment]
bot, build com.coeck_studios.Dinonuggys-Journey

File: com.coeck_studios.Dinonuggys-Journey.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline com.coeck_studios.Dinonuggys-Journey.appdata.xml:1]: Ideally, this should be maintained upstream.

File: com.coeck_studios.Dinonuggys-Journey.desktop [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```
Comment (barthalion) [inline com.coeck_studios.Dinonuggys-Journey.desktop:1]: Ideally, this should be maintained upstream.

**Comment by barthalion** [comment]
```
builddir/files/share/appdata/com.coeck_studios.Dinonuggys-Journey.appdata.xml: FAILED:
• style-invalid         : <caption> cannot end in '.' [At the start of the journey Dinonuggy is surrounded by kitchen utensils.]
• tag-missing           : <release> required
```

**Comment by barthalion** [comment]
bot, build com.coeck_studios.Dinonuggys-Journey

**Comment by barthalion** [comment]
```
builddir/files/share/appdata/com.coeck_studios.Dinonuggys-Journey.appdata.xml: FAILED:
• style-invalid         : <caption> cannot end in '.' [At the start of the journey Dinonuggy is surrounded by kitchen utensils.]
• tag-missing           : <release> required
```

File: com.coeck_studios.Dinonuggys-Journey.appdata.xml (hunk 1 above)
Comment (papojari) [reply com.coeck_studios.Dinonuggys-Journey.appdata.xml]: [Done](https://codeberg.org/CoEck/Dinonuggys-Journey/src/branch/main/com.coeck_studios.Dinonuggys-Journey.appdata.xml)

File: com.coeck_studios.Dinonuggys-Journey.desktop (hunk 2 above)
Comment (papojari) [reply com.coeck_studios.Dinonuggys-Journey.desktop]: [Done](https://codeberg.org/CoEck/Dinonuggys-Journey/src/branch/main/com.coeck_studios.Dinonuggys-Journey.desktop)

**Comment by papojari** [comment]
> 

I can't reproduce this locally if I rebase to 103c244f6cbd79c31364e8f8df4aaac19d5661b3. Validating and building both works on my machine. The errors spit out from the bot are fixed.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.coeck_studios.Dinonuggys-Journey.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+app-id: com.coeck_studios.Dinonuggys-Journey
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.godotengine.godot.BaseApp
+base-version: '3.4'
+sdk: org.freedesktop.Sdk
+command: godot-runner
```
Comment (papojari) [inline com.coeck_studios.Dinonuggys-Journey.yml:7]:
```suggestion
command: godot-runner
finish-args:
  - "--share=ipc"
  - "--socket=x11"
  - "--socket=pulseaudio"
  - "--device=dri"
```

File: com.coeck_studios.Dinonuggys-Journey.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        url: https://codeberg.org/CoEck/Dinonuggys-Journey/archive/v0.55.0.tar.gz
+        sha256: a93a3e73789ecd2cb0fc27223e8625bd5767679804550aeae2eac3e38697059a
+      - type: file
+        url: https://codeberg.org/attachments/5d46e8fd-9e6b-4ef0-bdee-ce97168ba3b1
+        sha256: 0aae94ef1974c657b693317d3b8367504bf03ccfd515300cb63f4ab5afe38d12
+      - type: file
+        path: com.coeck_studios.Dinonuggys-Journey.appdata.xml
+      - type: file
+        path: com.coeck_studios.Dinonuggys-Journey.desktop
+finish-args:
```
Comment (papojari) [inline com.coeck_studios.Dinonuggys-Journey.yml:28]:
```suggestion
```

File: com.coeck_studios.Dinonuggys-Journey.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  - "--share=ipc"
```
Comment (papojari) [inline com.coeck_studios.Dinonuggys-Journey.yml:29]:
```suggestion
```

File: com.coeck_studios.Dinonuggys-Journey.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
```
Comment (papojari) [inline com.coeck_studios.Dinonuggys-Journey.yml:32]:
```suggestion
```

File: com.coeck_studios.Dinonuggys-Journey.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      - type: file
+        url: https://codeberg.org/attachments/5d46e8fd-9e6b-4ef0-bdee-ce97168ba3b1
+        sha256: 0aae94ef1974c657b693317d3b8367504bf03ccfd515300cb63f4ab5afe38d12
+      - type: file
+        path: com.coeck_studios.Dinonuggys-Journey.appdata.xml
+      - type: file
+        path: com.coeck_studios.Dinonuggys-Journey.desktop
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
```
Comment (papojari) [inline com.coeck_studios.Dinonuggys-Journey.yml:30]:
```suggestion
```

File: com.coeck_studios.Dinonuggys-Journey.yml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  - "--socket=pulseaudio"
```
Comment (papojari) [inline com.coeck_studios.Dinonuggys-Journey.yml:31]:
```suggestion
```

**Comment by papojari** [comment]
Ok I know how to do multi-line code suggestions now for next time.

**Comment by papojari** [comment]
bot, build com.coeck_studios.Dinonuggys-Journey

**Review by hfiguiere — APPROVED** [review]

**Comment by papojari** [comment]
I thought you would create a new repository @hfiguiere.

**Comment by hfiguiere** [comment]
I can't. Some of the admins will sometime soon unless they have comments.

**Pull request closed without merging**